docs/topics/kmm/kmm-getting-started.md

```diff
@@ -1,22 +1,54 @@
-[//]: # (title: Getting started)
+[//]: # (title: Get started with Kotlin Multiplatform Mobile)
+[//]: # (description: cross platform mobile development, android vs ios development)
```

Comment:
`description` should be a summary of 155-160 characters that motivates people to click this search result. Please let me know if you need help here.